tests/manage/pv_services/test_delete_project_with_pvc_and_pods.py

```diff
@@ -0,0 +1,165 @@
+"""
+The purpose of this test case is to delete at least 2 projects:
+One project with just pvcs and the other with pvcs attached to a pod
+"""
+import logging
+import random
+import time
+import pytest
+
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.utility.retry import retry
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+
+@retry(UnexpectedBehaviour, tries=10, delay=3, backoff=1)
+def check_ceph_used_space():
```

Comment:
This looks like an infra material, not just for this test?

Reply:
Indeed. This function could be useful for other tests. Could you please move it to be a method in the CephCluster class here in ocs_ci.ocs.cluster.CephCluster?